mne/utils/docs.py

```diff
@@ -728,6 +728,50 @@
     If True, use a linear transparency between fmin and fmid.
     None will choose automatically based on colormap type.
 """
+
+# STC label time course
+docdict['eltc_labels'] = """
+labels : Label | BiHemiLabel | list of Label or BiHemiLabel
+    The labels for which to extract the time course.
+"""
+docdict['eltc_src'] = """
+src : list
+    Source spaces for left and right hemisphere.
+"""
+docdict['eltc_mode'] = """
+mode : str
+    Extraction mode, see Notes.
+"""
+docdict['eltc_allow_empty'] = """
+allow_empty : bool
+    Instead of emitting an error, return all-zero time courses for labels
+    that do not have any vertices in the source estimate. Default is ``False``.
+"""
+docdict['eltc_mode_notes'] = """
+Valid values for ``mode`` are:
+
+``'max'``
+    Maximum value across vertices at each time point within each label.
+``'mean'``
+    Average across vertices at each time point within each label. Ignores
+    orientation of sources.
+``'mean_flip'``
+    Finds the dominant direction of source orientations within each label,
```

Comment:
Somewhere in here it should say some variant of "based on the source space normal directions" to make it clear that the "dominant direction" is entirely based on anatomy (not the time series data in `stc`)